website/pages/docs/agent/caching/index.mdx

```diff
@@ -49,6 +49,14 @@ configuration will be overridden if the request already has a token attached,
 in which case, the token present in the request will be used to forward the
 request to the Vault server.
 
+## Forcing Auto-Auth Token
+
+Vault Agent can be configured to force the use of the auto-auth token by using
+the value `force` for the `use_auto_auth_token` option. This configuration
+overrides the default behavior described above in [Using Auth-Auth
+Token](/docs/agent/caching#using-auto-auth-token), and instead ignores any
+existing Vault token in the request and instead uses the auth-auth token. 
+
```

Comment:
What is the behavior is `force` is set, there is no auto-auth token, but the request does contain a token?

Reply:
This "can't" happen; if the configuration has `use_auto_auth_token` set to `true` or `force`, and no valid `auto_auth` block is defined, then Vault Agent will exit with status code `1` at startup

Reply:
👍